package.json

```diff
@@ -14,16 +14,19 @@
     "@types/hapi__joi": "^15.0.1",
     "enjoi": "^6.0.0",
     "moment": "^2.23.0",
-    "mysql2": "^1.6.5"
+    "mysql2": "^1.6.5",
+    "sqlite3": "^4.0.9"
```

Comment:
We use this only in tests -> should this be in devDeps?

Reply:
Good spot, moved to `devDependencies`.